app/views/signals/reservation.html.erb

```diff
@@ -22,7 +25,7 @@
       <h2 class="reservation-section__title">FAQ</h2>
 
       <div class="signal-faqs">
-        <%= react_component_with_jss "SignalFaqPanel",
+        <%= react_component "SignalFaqPanel",
```

Comment:
I'm assuming changes to react_component_with_jss and prerender: true are unintentional?

Reply:
Thanks for checking @anthotsang, this was intentional; I'm temporarily rolling back since the SSR JSS is causing some visual weirdness and we need to get this out ASAP.